### ISSUE TITLE
Identify if photo has an any obstructions in face area (e.g.: mask, sunglasses, hand covering face)

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#950]
Identify if photo has an any obstructions in face area (e.g.: mask, sunglasses, hand covering face)

**Comment by priyanka31-12** [comment]
Hi, can you brief me about what all do I need to know to contribute in this issue?

**Comment by akshitagupta15june** [comment]
yes means you to identify in a face image like what objects they are wearing like sunglasses so u should label them near the object

**Comment by hemanthraj2001** [comment]
Hello, @akshitagupta15june 
I would like to work on this issue! Please assign it to me!

**Comment by akshitagupta15june** [comment]
ok

**Comment by SURABHI-GUPTA** [comment]
@akshitagupta15june I would like to work on this.

**Comment by akshitagupta15june** [comment]
assigning you @SURABHI-GUPTA

**Comment by SURABHI-GUPTA** [comment]
> yes means you to identify in a face image like what objects they are wearing like sunglasses so u should label them near the object

So, do we need to create annotations like bounding boxes for obstructions present in the image? If yes, on what dataset are we supposed to do that?

**Comment by salmeenessam** [comment]
I would like to be assigned to this issue

**Comment by akshitagupta15june** [comment]
Assigning you

**Comment by shireenchand** [comment]
For this do we even consider specs as an obstruction?

**Comment by learner2102** [comment]
Anyone working on this issue?
